warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reload/ReloadFilesFromLoadedJob.java

```diff
@@ -0,0 +1,130 @@
+package datawave.ingest.mapreduce.job.reload;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.FileUtil;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.LongWritable;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.mapreduce.Job;
+import org.apache.hadoop.mapreduce.Mapper;
+import org.apache.hadoop.mapreduce.lib.input.NLineInputFormat;
+import org.apache.hadoop.mapreduce.lib.output.NullOutputFormat;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ReloadFilesFromLoadedJob {
+    public static class ReloadFilesFromLoadedMapper extends Mapper<LongWritable,Text,Text,Text> {
+
+        protected final Logger log = LoggerFactory.getLogger(getClass());
+
+        @Override
+        protected void map(LongWritable key, Text value, Context context) throws IOException, InterruptedException {
+            log.info("Processing file: \"{}", value.toString());
+
+            String[] parts = value.toString().split("/");
+            String datatype = parts[3];
+            String year = parts[4];
+            String month = parts[5];
+            String day = parts[6];
+            String hour = parts[7];
+            String filename = parts[8];
+
+            String dirPath = "/data/loaded/archive/" + datatype + "/" + year + "/" + month + "/" + day + "/";
```

Comment:
These should be configurable and align with what is in the FlagConfigs.  Not every system has a baseDir of /data, some differ.